google-cloud-bigquerystorage/src/main/java/com/google/cloud/bigquery/storage/v1/Exceptions.java

```diff
@@ -78,6 +78,27 @@ protected SchemaMismatchedException(Status grpcStatus, Metadata metadata, String
     }
   }
 
+  /** Offset already exists. */
+  public static final class OffsetAlreadyExists extends StorageException {
```

Comment:
could we make this exception richer by providing the actual and expected offset? It can be done by parsing the error message.

Reply:
done